src/pages/place/[placeId]/status.js

```diff
@@ -1,55 +1,173 @@
 import { useRouter } from 'next/router';
 import { QueryStatus } from '../../../hooks/api/authenticated-query';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import { getApplicationServerSideProps } from '../../../utils/getApplicationServerSideProps';
 import { Alert, AlertVariants } from '../../../components/publiq-ui/Alert';
 import { Button, ButtonVariants } from '../../../components/publiq-ui/Button';
 import { Spinner } from '../../../components/publiq-ui/Spinner';
+import { RadioButtonGroup } from '../../../components/publiq-ui/RadioButtonGroup';
 import { Page } from '../../../components/publiq-ui/Page';
+import { TextAreaWithLabel } from '../../../components/publiq-ui/TextAreaWithLabel';
 import { useTranslation } from 'react-i18next';
-import { useGetPlaceById } from '../../../hooks/api/places';
+import { useChangeStatus, useGetPlaceById } from '../../../hooks/api/places';
+import { Text } from '../../../components/publiq-ui/Text';
+import { getValueFromTheme } from '../../../components/publiq-ui/theme';
+import { Stack } from '../../../components/publiq-ui/Stack';
+import { Inline } from '../../../components/publiq-ui/Inline';
+import { dehydrate } from 'react-query/hydration';
+
+const getValue = getValueFromTheme('statusPage');
+
+const OfferStatus = {
```

Comment:
Some constants to put in a more general file and to reuse?

Reply:
In the following PR I have moved OfferStatus and maxReasonLength to EventStatusModal and import them from there. But I'm also not sure about that approach. The thing is we've tried so far to define constants in logical places and tried to avoid putting them in a constants directory.

Reply:
The logical place is fine, as long as it gets reused.